src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller.car;

```

Comment:
another feature, controllers are usually not separated by packages 
(you can not fix it now, but take it into account when designing your project)